kiwi/schema/kiwi.rnc

```diff
@@ -970,6 +982,25 @@ div {
         attribute sourcetype {
             "baseurl" | "metalink" | "mirrorlist"
         }
+    k.repository.use_for_bootstrap.attribute =
+        ## Specify whether this repository should be the one used by de
+        ## debootstrap or no. False by default. Only a single repository
```

Comment:
Please remove references to debootstrap.

Reply:
Those are left on purpose since they are pure literals for the user and currently this is only being used to workaround debootstrap limitations.

Reply:
as a generic property that we can extend elsewhere, we should not have those assumptions exposed.

Reply:
ok updated